exercises/nodejs/src/game.js

```diff
@@ -1,232 +1,208 @@
-    scienceQuestions.push('Science Question ' + questionIndex)
-    sportsQuestions.push('Sports Question ' + questionIndex)
-    rockQuestions.push(this.createRockQuestion(questionIndex))
+    popQuestions.push("Pop Question " + questionIndex);
+    scienceQuestions.push("Science Question " + questionIndex);
+    sportsQuestions.push("Sports Question " + questionIndex);
+    rockQuestions.push(this.createRockQuestion(questionIndex));
   }
 
   this.isPlayable = function (howManyPlayers) {
-    return howManyPlayers >= 2
-  }
+    return howManyPlayers >= 2;
+  };
 
   this.addPlayer = function (playerName) {
-    players.push(playerName)
-    playerPositions[this.howManyPlayers() - 1] = 0
-    playerPurses[this.howManyPlayers() - 1] = 0
-    inPenaltyBox[this.howManyPlayers() - 1] = false
+    players.push(playerName);
+    playerPositions[this.howManyPlayers() - 1] = 0;
+    playerPurses[this.howManyPlayers() - 1] = 0;
+    inPenaltyBox[this.howManyPlayers() - 1] = false;
 
-    console.log(playerName + ' was added')
-    console.log('They are player number ' + players.length)
+    console.log(playerName + " was added");
+    console.log("They are player number " + players.length);
 
-    return true
-  }
+    return true;
+  };
 
   this.howManyPlayers = function () {
-    return players.length
-  }
+    return players.length;
+  };
 
   var askQuestion = function () {
     if (currentCategory() === POP_CATEGORY) {
-      console.log(popQuestions.shift())
+      console.log(popQuestions.shift());
     }
     if (currentCategory() === SCIENCE_CATEGORY) {
-      console.log(scienceQuestions.shift())
+      console.log(scienceQuestions.shift());
     }
     if (currentCategory() === SPORT_CATEGORY) {
-      console.log(sportsQuestions.shift())
+      console.log(sportsQuestions.shift());
     }
     if (currentCategory() === ROCK_CATEGORY) {
-      console.log(rockQuestions.shift())
+      console.log(rockQuestions.shift());
     }
-  }
+  };
 
   this.roll = function (roll) {
-    console.log(players[currentPlayer] + ' is the current player')
-    console.log('They have rolled a ' + roll)
+    console.log(players[currentPlayer] + " is the current player");
+    console.log("They have rolled a " + roll);
 
     if (inPenaltyBox[currentPlayer]) {
       if (roll % 2 != 0) {
-        isGettingOutOfPenaltyBox = true
+        isGettingOutOfPenaltyBox = true;
 
         console.log(
-          players[currentPlayer] + ' is getting out of the penalty box'
-        )
-        playerPositions[currentPlayer] = playerPositions[currentPlayer] + roll
+          players[currentPlayer] + " is getting out of the penalty box"
+        );
+        playerPositions[currentPlayer] = playerPositions[currentPlayer] + roll;
         if (playerPositions[currentPlayer] > 11) {
-          playerPositions[currentPlayer] = playerPositions[currentPlayer] - 12
+          playerPositions[currentPlayer] = playerPositions[currentPlayer] - 12;
```

Comment:
This block is repeated a few times in the `roll` function, maybe this could be moved to its own function?